src/System.Windows.Forms/src/System/Windows/Forms/ListViewGroup.cs

```diff
@@ -172,6 +186,29 @@ public HorizontalAlignment FooterAlignment
             }
         }
 
+        internal bool Focused { get; set; }
+
+        private bool IsDefault
+        {
+            get
+            {
+                if (ListView?.Groups is null)
+                {
+                    return false;
+                }
+
+                foreach (ListViewGroup group in ListView.Groups)
+                {
+                    if (group == this)
+                    {
+                        return false;
+                    }
+                }
+
+                return true;
```

Comment:
`Groups` will never be **null**. 
I think `Contains()` is more readable here
```suggestion
                if (ListView is null || ListView.Groups.Contains(this))
                {
                    return false;
                }

                return true;
```

Reply:
Let's go further:
```cs
return ListView != null && !ListView.Groups.Contains(this);
```

Reply:
Yes, because this is a ListViewGroup.IsDefault

Reply:
Haha 🤦‍♂️